docs/misc/assumptions.md

```diff
@@ -4,11 +4,10 @@
 
 ScubaGear has been tested against tenants that have an M365 E3 or G3 and E5 or G5 license bundle. It may still function for tenants that do not have one of these bundles.
 
-Some of the policy checks in the baseline rely on the following licenses which are included by default in M365 E5 and G5.
+> **Note**: DOD endpoints are included, but have not been tested. Please open an issue if you encounter bugs.
 
+Some of the policy checks in the baselines rely on the following licenses, which are included by default in M365 E5 and G5:
```

Comment:
They shouldn't be mandated to have Purview Audit (Premium) in practice anymore because of the changes in the log types included in Purview (standard), which is observed in the note for MS.DEFENDER.6.2v1

As for retention, they may choose to retain it in Microsoft using purview or use a third party solution (Splunk, etc) in their SOC.  Both would meet the policy, as is explained in the note for that policy.

Reply:
If we don't count Purview Audit (Premium), "almost all cases...can be met using a third-party service" become "all cases." That was the one exception. I'll take out the "almost."

Reply:
What about MS.AAD.2.{1-3}v1 required AAD P2 as well?  Agree there are not many.

Reply:
You're right, I wasn't tracking that one.

Reply:
For tracking purposes in [MS.AAD.4.1v1](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/baselines/aad.md#license-requirements-3) -- we say an Azure subscription may be required for forwarding logs to a SIEM, although which SIEM an agency uses falls under the 3rd-party service category.